ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/CommandSubmitter.scala

```diff
@@ -101,6 +119,14 @@ case class CommandSubmitter(
     )
   }
 
+  private def allocateDivulgeeParties(divulgeePartyNames: Seq[String])(implicit
```

Comment:
This appears to be identical to allocateObserverParties.  Could there be a single method `allocateParties` that is used for both purposes.

Reply:
Ok